### PR TITLE
Change working dir to /tmp in vmagent chart

### PR DESCRIPTION
**Pull request by ulikl**
"/" may not be writable in restricted containers

### CODE REVIEW

File: charts/victoria-metrics-agent/templates/deployment.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@ spec:
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
+          workingDir: /tmp
```
Comment (tenmozes) [inline charts/victoria-metrics-agent/templates/deployment.yaml:42]: this is not a backward compatible changes, so please move `workingDir` to values.yaml (the default value should be "/") and bump chart version

Comment (ulikl) [reply charts/victoria-metrics-agent/templates/deployment.yaml]: you are right. I adapted it.

**Comment by tenmozes** [comment]
thank you for contribution, will do release soon

**Comment by tenmozes** [comment]
the new version of helm chart for vmagent (0.4.9) contains `containerWorkingDir` variable

**Pull request merged**